apps/README.md

```diff
@@ -0,0 +1,26 @@
+`FluentUI Tester` is the test app that we use to test our FluentUI components during development.
```

Comment:
Could we make a mention to the fact that it uses [react-native-test-app](https://github.com/microsoft/react-native-test-app) to generate the test app? Just so that developers can refer to documentation there as well. Also once again many thanks to @tido64 and the rest of his team for helping with this :D

Reply:
Also, I thought it was fluent-tester, we should probably make a PR to standardize that name, I'm pretty sure I still see Fabric Tester in code :D

Reply:
Discussed offline, we're going to keep the name "FluentUI Tester", since the component libraries are part of the FluentUI family.